src/command/remark.ts

```diff
@@ -308,9 +316,13 @@ export enum RemarkType {
   SnowDepth = "SnowDepth",
   SunshineDuration = "SunshineDuration",
   WaterEquivalentSnow = "WaterEquivalentSnow",
+
+  // Canada commands below
+  NextForecastBy = "NextForecastBy",
 }
 
-export type Remark =
+// Remark types that are not date based
+export type RemarkBase =
```

Comment:
Remove export

Reply:
I was thinking of not adding `export` when doing this but I guess I forgot to actually not add it...